c++/src/kj/exception.h

```diff
@@ -440,6 +440,13 @@ kj::ArrayPtr<void* const> computeRelativeTrace(
 //
 // This is useful for debugging, when reporting several related traces at once.
 
+class ErrorHandler {
+  // A sink for Exceptions. Implement this interface when you need to provide custom error handling
+  // for generic code.
+public:
+  virtual void taskFailed(kj::Exception&& exception) = 0;
```

Comment:
The name `taskFailed()` is kind of specific to `TaskSet`. I'm not sure this is the right interface to make more general in this way.

In fact, I regret defining this interface for `TaskSet`. I think I should have had it take a `kj::Function<void(kj::Exception&&)>` to the constructor instead.

Can we do that instead, for the TLS error handler callback?

Reply:
Yeah, happy to do so!